source/install/mattermost-omnibus.rst

```diff
@@ -0,0 +1,117 @@
+Installing Mattermost Omnibus
+============================
+
+Overview
+---------
+
+Mattermost Omnibus is a Debian package that packages the individual components of a Mattermost deployment into a single installation process. The package leverages the ``apt`` package manager to install and keep the components of the platform updated, using a custom CLI and ansible recipes to link those components together and configure them.
+
+Mattermost Omnibus currently supports Ubuntu's ``bionic`` and ``focal`` distributions. Support for RedHat/CentOS distributions will be available in future versions. 
+
+Prerequisites
+-------------
+
+- A clean Ubuntu server (18.04. Or 20.04)
+- Root level access to the server
+
+.. note:: 
+
+  With Mattermost Omnibus, the ``config.json`` file no longer exists. So you’ll need to use mmctl to make changes to your Mattermost server configuration using ``mmctl config edit``. If you wish to manage your server locally, you may want to enable local mode for mmctl.
+
+Installing Mattermost Omnibus
+------------------------------
+
+During installation you can provide an existing domain name that points to the Ubuntu server or, if you don't have one available, you can log in using the server's IP address.
+
+.. note::
+  Plugin uploads and HTTPS are enabled by default. These settings are modified in the ``yaml`` file using `mmctl config edit <https://docs.mattermost.com/administration/mmctl-cli-tool.html#mmctl-config-edit>`__. 
+
+The Omnibus repositories are configured using a cURL command: 
+
+``curl -o- deb.packages.mattermost.com/repo-setup.sh | sudo bash``
+
+This command installs and configures a ``PostgreSQL`` database, an ``nginx`` web server to act as a proxy, and ``certbot`` to issue and renew the SSL certificate. 
+
+In order to issue that certificate, the installer requests a domain name and an email address from us. These are used to generate the certificate and deliver any related communications respectively.
+
+Installing Omnibus Package and Mattermost Server
+------------------------------------------------
+
+If you're installing Omnibus, and have a domain and SSL use: 
+
+``sudo apt install mattermost-omnibus``.
+
+After all the packages are installed, Omnibus runs the ansible scripts that configure all the platform components and starts the server. To access Mattermost, open a browser, navigate to your domain, and create the System Admin user to start using the platform. 
+
+You can also install Mattermost without a domain name or SSL (not recommended). To do this use: 
+
+``sudo MMO_HTTPS=false apt install mattermost-omnibus``
+
+Updating Mattermost Omnibus
+-----------------------------
+
+Mattermost Omnibus is integrated with Ubuntu’s OS's package manager. When a new Mattermost version is released, run:
+
+``sudo apt update && sudo apt upgrade``
+
+This downloads the latest version of Mattermost and updates your Mattermost platform. 
+
+Removing Mattermost Omnibus
```

Comment:
Adding here a suggestion for a section that explains the different config properties that the `mmomni.yml` file has, and what are they useful for:

```suggestion
Configuring Mattermost Omnibus
----------------------------------------

Omnibus has its own configuration file located in ``/etc/mattermost/mmomni.yml``. This file contains the data that Omnibus needs to configure the platform and connect all the services together. For Omnibus to work properly, there are some configuration parameters that are fixed and cannot be changed through the web interface (for example, the port that Mattermost uses to run) and some others that need to be configured directly through this file instead of in the Mattermost web interface or the configuration. The properties that you can configure in this file are:

``db_user``: the PostgreSQL database user. This value is generated during the Omnibus installation and should not be changed.
``db_password``: the PostgreSQL database password. This value is generated during the Omnibus installation and should not be changed.
``fqdn``: the domain name for the Mattermost application. This is the value that is asked during the install process, and it’s used to populate the ServiceSettings.SiteURL Mattermost configuration property, as well as to retrieve and configure the SSL certificate for the server.
``email``: the email for certificate communications. This is the value that is asked during the install process, and it will not be used if https is disabled.
``https``: this indicates if the platform should be configured to use https or http. It can have the ``true`` and ``false`` values. The recommended way to install Mattermost is to use https, but you can disable it if required. You can read more about this here [[LINK TO THE FAQ]]
``data_directory``: this is the directory where Mattermost stores its data
``enable_plugin_uploads``: this setting is used to configure the ``PluginSettings.EnableUploads`` Mattermost configuration property. It can have the ``true`` and ``false`` values
``enable_local_mode``: this setting is used to configure the ``ServiceSettings.EnableLocalMode`` Mattermost configuration property. It can have the ``true`` and ``false`` values

Removing Mattermost Omnibus
```